From ffc5d9dda91258539280b3af78c3b04f8c20cc5b Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Thu, 3 Aug 2017 12:10:22 +0800
Subject: [PATCH 2279/5242] MLK-16119-05: dts: Disabled hdmi property

commit  d14f42f762fd7d608928ee76385a942662918dcd from
https://source.codeaurora.org/external/imx/linux-imx.git

HDMI driver loading depend on HDMI FW.
But not all boards default included HDMI FW.
So disabled hdmi property in dts now.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 6a9642f..b2fd27a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -407,7 +407,7 @@
 
 &hdmi {
 	disp-dev = "hdmi_disp";
-	status = "okay";
+	status = "disabled";
 };
 
 &i2c2 {
-- 
1.7.9.5

